tests/phpunit/tests/fonts/font-face/wpFontFaceResolver/getFontsFromThemeJson.php

```diff
@@ -37,20 +37,62 @@ public function test_should_return_all_fonts_from_theme() {
 		$this->assertSame( $expected, $actual );
 	}
 
+	/**
+	 * @ticket 60605
+	 */
+	public function test_should_return_all_fonts_from_all_theme_origins() {
+		switch_theme( static::FONTS_THEME );
+
+		// Add a filter to add fonts to the theme.json custom origin.
+		add_filter(
+			'wp_theme_json_data_theme',
+			static function ( $theme_json_data ) {
+				$data = $theme_json_data->get_data();
+
+				// Add font families to the custom origin of theme json.
+				$data['settings']['typography']['fontFamilies']['custom'] = $this->get_custom_font_families( 'input' );
```

Comment:
Whoopsie, the function does not understand `$this`.

How to fix? Move this function callback to be a method instead.

Reply:
What about this way?https://github.com/WordPress/wordpress-develop/pull/6161/commits/fea390858e2e0e58fb170795360c868e2d165844